hr_timesheet_sheet/models/account_analytic_line.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class AccountAnalyticLine(models.Model):
+    _inherit = 'account.analytic.line'
+
+    sheet_id_computed = fields.Many2one(
```

Comment:
`computed_sheet_id`? Also, would be nice to have a comment explaining the purpose of this field (compared w/ the next one)

Reply:
I think it's better as is, for having the same name beginning

Reply:
This is the same that v10. Here I didn't do any change.